libctru/source/allocator/vram.cpp

```diff
@@ -8,21 +8,28 @@ extern "C"
 #include "mem_pool.h"
 #include "addrmap.h"
 
-static MemPool sVramPool;
+static MemPool sVramPoolA;
+static MemPool sVramPoolB;
 
 static bool vramInit()
 {
-	auto blk = MemBlock::Create((u8*)0x1F000000, 0x00600000);
-	if (blk)
+	auto blkA = MemBlock::Create((u8*)0x1F000000, 0x00300000);
+	if (blkA)
 	{
-		sVramPool.AddBlock(blk);
-		rbtree_init(&sAddrMap, addrMapNodeComparator);
-		return true;
+		auto blkB = MemBlock::Create((u8*)0x1F300000, 0x00300000);
+		if(blkB)
+		{
+			sVramPoolA.AddBlock(blkA);
+			sVramPoolB.AddBlock(blkB);
+			rbtree_init(&sAddrMap, addrMapNodeComparator);
+			return true;
+		}
+		free(blkA);
 	}
 	return false;
 }
 
-void* vramMemAlign(size_t size, size_t alignment)
+void* vramBankMemAlign(VRAM_ALLOCATOR bank, size_t size, size_t alignment)
 {
```

Comment:
Can the old `vramMemAlign` be restored, working in a similar way to `vramAlloc`?